dags/bqetl_backfill_initiate.py

```diff
@@ -44,8 +47,22 @@
 
     @task_group
     def initiate_backfill(backfill):
+        def invite_watchers_to_slack_channel(entry):
```

Comment:
If it doesn't work as is, I think it might end up having to be a task 🤔 We could also encapsulate this in a subclass of `SlackAPIOperator`